install/kubernetes/helm/istio/charts/security/values.yaml

```diff
@@ -14,6 +14,9 @@ citadelHealthCheck: false
 # 90*24hour = 2160h
 workloadCertTtl: 2160h
 
+# Citadel default behavior if the ca.istio.io/env namespace label not found
```

Comment:
I'm not sure the name is right. We don't typically use the 'by default' in names - the value is the actual default. 

Also not clear - what does it mean to 'enable namespace' ? For what ? There are many things we use namespaces for.

Maybe something like 'generateSecretsByLabel' ? or 'requireNamespaceLabel ?

Reply:
Also please expand the comment with an example, and anything you can think of - this is what gets auto-generated in docs and what users will actually use to decide how to set it.

Reply:
This is the exact same name we use for sidecar injector for the same thing. I don't think the name is great, but don't have any better suggestions :slightly_smiling_face: 

adding some more notes like "if true, citadel will generate secrets for the namespace without the label ....." would be useful

Reply:
I agree this name is not great.
Just one thought, `generateSecretForUnlabeledNamespaces` to be more precise?

Reply:
Agreed that the name should be more descriptive. I chose `enableNamespacesByDefault` for consistency with the [sidecar injector API](https://istio.io/docs/reference/config/installation-options/#sidecarinjectorwebhook-options), but if we're willing to break that consistency, something like `generateSecretForUnlabeledNamespaces` would be better.